src/Backend/Modules/Locale/Engine/LocaleAnalyser.php

```diff
@@ -22,7 +24,47 @@ public function __construct(string $application, string $modulesPath, array $ins
 
     public function findMissingLocale(string $language): array
     {
+        $moduleFiles = $this->findModuleFiles();
+
         // @TODO
         return [];
     }
+
+    /**
+     * Get a string between two key strings
+     *
+     * @param string $start front key string
+     * @param string $end back key string
+     * @param string $str the string that needs to be checked
+     *
+     * @return mixed return string or false
+     */
+    private function getInBetweenStrings(string $start, string $end, string $str)
+    {
+        $matches = [];
+        preg_match_all("@$start([a-zA-Z0-9_]*)$end@", $str, $matches);
+
+        return isset($matches[1]) ? current($matches[1]) : '';
+    }
+
+    private function findModuleFiles(): array
```

Comment:
Wouldn't it be more logical to loop all modules and fetch each's files? I know that ignores modules that aren't installed but it seems to make more sense

Reply:
I think it would make the code more complex